src/Feersum.CompilerServices/Compile/Compiler.fs

```diff
@@ -1234,7 +1234,7 @@ module Compilation =
 
         let ast, diagnostics =
             let nodes, diagnostics =
-                List.map Parse.parseFile sources
+                List.map LegacyParse.parseFile sources
                 |> List.fold (fun (nodes, diags) (n, d) -> (List.append nodes [ n ], List.append d diags)) ([], [])
 
             { Location = TextLocation.Missing
```

Comment:
We probably want to come up with a better way of modelling this. The previous parser was run on each input, and then we pasted them together into a single big SEQ. Instead I guess we should have a wrapper `Compilation` type, which contains a set of `BoundCompilationUnits`. We can bind them in sequence, and propagate some state (lib definitions etc.). The key is we _don't_ want to have public items from one unit just become 'magically' available in another. This will probably want to be different for script compilations however. In that case we'd _want_ multiple passes at compilation to share definitions from the previous.

We _may_ want to handle this by just having the last compilation unit in the compilation be treated specially and allow definitions from that to leak into the root binder scope. That way inherited compilations from scripting would have access to the previous definitions.